flake.nix

```diff
@@ -115,15 +115,17 @@
             buildInputs =
               [
                 benchmark
-                launcher
-                router
-                server
+                cargo
                 client
+                clippy
+                ffmpeg
+                launcher
                 openssl.dev
                 pkg-config
-                cargo
+                router
+                rustPlatform.bindgenHook
```

Comment:
No bindgenhook.

ffmpeg is ok, but please let's keep the diff readable (no sorting).